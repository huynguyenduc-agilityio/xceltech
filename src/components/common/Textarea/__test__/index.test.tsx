import { render } from '@testing-library/react';

// Components
import { Textarea } from '..';

describe('Textarea component', () => {
  it('should match snapshot', () => {
    const { container } = render(<Textarea placeholder="Description" />);

    expect(container).toMatchSnapshot();
  });
});
