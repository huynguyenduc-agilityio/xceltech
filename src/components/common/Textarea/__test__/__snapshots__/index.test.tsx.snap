// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea component should match snapshot 1`] = `
<div>
  <textarea
    class="flex min-h-[80px] w-full rounded-regular bg-blue-light px-5 py-4 text-lg text-black-smoky placeholder:text-gray-soft outline-none resize-none disabled:cursor-not-allowed disabled:opacity-50"
    placeholder="Description"
  />
</div>
`;
